### PR TITLE
æneria v2

### PR DESCRIPTION
**Pull request by SimonMellerin**
## Problem

- Update/Adapt to major version 2 of aeneria 

## Solution

- *And how do you fix that problem*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Review by alexAubin — APPROVED** [review]

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

**Comment by SimonMellerin** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -32,24 +33,19 @@ ram.runtime = "50M"
     [install.domain]
     type = "domain"
 
-    [install.path]
-    type = "path"
-    default = "/aeneria"
```
Comment (sereinity) [inline manifest.toml:37]: Weird, we can still change the path after the upgrade.

File: manifest.toml [hunk 2]
```diff
@@ -2,10 +2,10 @@ packaging_format = 2
 
 id = "aeneria"
 name = "æneria"
-description.en = "Dashboard to analyse your energy consumption data from Linky & weather"
-description.fr = "Tableau de bord pour analyser votre consomation d'énergie à partir des données d'un Linky et de la météo"
+description.en = "Dashboard to analyse your energy consumption data from Linky, Gazpar & weather"
+description.fr = "Dashboard pour analyser vos données de consommation d'énergie Linky, Gazpar et météo"
 
-version = "1.1.6~ynh5"
+version = "2.0.3~ynh1"
```
Comment (sereinity) [inline manifest.toml]: Don't forget to update this (as in READMEs)

Comment (SimonMellerin) [reply manifest.toml]: :+1:

File: manifest.toml (hunk 1 above)
Comment (SimonMellerin) [reply manifest.toml:37]: yep, I saw that too, but if you try to put a path, I think it fails

**Pull request merged**